config/filter.d/nextcloud-auth.conf

```diff
@@ -0,0 +1,18 @@
+# Fail2Ban filter file for Nextcloud login failures
+#
+# Author: Eric Wolf
```

Comment:
I assumed that the author is the person who created the filter file.
Since the regex is from the hardening guide (which after some investigation turns out to be based on #2485) I would put your name there with the name of the person who added the trusted domain error extension in nextcloud/documentation#2421.